Makefile

```diff
@@ -1,3 +1,7 @@
+REGISTRY         = docker.io/ansibleplaybookbundle
```

Comment:
Can you break up these two into ```$(Registry)/$(Project)```.  That will allow anyone to point this at any registry and change the name if they want.